boa3/model/builtin/native/nativecontract.py

```diff
@@ -10,11 +11,13 @@ class NativeContract:
 
     # Interop Types
     BlockType = BlockType.build()
+    RoleType = RoleType()
     TransactionType = TransactionType.build()
 
     # Class Interfaces
     Ledger = LedgerClass()
     Policy = PolicyClass()
+    Role = RoleManagementClass()
```

Comment:
```suggestion
    RoleManagement = RoleManagementClass()
```
for better documenting the code, use the full name of the native contracts